src/css/tabs/osd.css

```diff
@@ -517,16 +527,31 @@ button {
   }
 }
 
-.tab-osd .grid-col.col4:first-child, .tab-osd  .grid-col.col4:last-child {
-    width: calc((100% - 390px) / 2);
+.tab-osd .grid-row {
+    justify-content: center;
+    gap: 7px;
+}
+
+.tab-osd .grid-row .elements {
+  flex: 1 1 auto;
+  width: 210px;
+  max-width: 700px;
 }
 
-.tab-osd .grid-row {
-    margin-bottom: 40px;
+.tab-osd .grid-row .osd-preview {
+  flex: 2 0 auto;
+  width: 350px;
+  max-width: max(min(950px,calc((100vh - 470px) * 1.25)),350px);
```

Comment:
```suggestion
  max-width: max(min(950px, calc((100vh - 470px) * 1.25)), 350px);
```

Reply:
Can you explain? This is not working? It does not adjust width (and height accordingly) when making the height of the window smaller?

Reply:
Indeed with smaller height the effect is noticed.